src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -517,6 +513,24 @@ private static String resolveVariables(String nodeValue, Collection<String> vari
         return resolved;
     }
 
+    private static String getPropertyValue(String propertyName) {
```

Comment:
The behavior to resolve the property is the same here regardless of whether it is prefixed by `env.` or not. It would be more concise to check for the prefix first, strip it if necessary, then apply the property resolution.

Reply:
But what if they happened to define a variable that starts with `env.`? I don't think anything prevents that, and that should be resolved before attempting to resolve the variable without the prefix.

Reply:
I would expect that if the `env` prefix is reserved as an indicator for environment variables then a user shouldn't be allowed to define non-environment variables that start with that prefix. We should probably find out how this is handled in Liberty so we can ensure consistent behavior.

Reply:
Documenting that @cherylking ensured via testing that non-environment variables can use the `env` prefix.